requirements.txt

```diff
@@ -1,28 +1,29 @@
+# Automatically generated by gen_requirements_all.py, do not edit
 -c homeassistant/package_constraints.txt
 
 # Home Assistant Core
+PyJWT==2.5.0
 aiohttp==3.8.1
```

Comment:
Sorting went wrong?

Reply:
No, strictly speaking. 
```
>>> 'P' < 'a'
True
```

Reply:
Right, lets make it human sorted in that case

Reply:
Done. This grew the diff (in generated files) by quite a lot since the *all files weren't human-sorted either.